pkgs/top-level/release-outpaths-parallel.nix

```diff
@@ -0,0 +1,56 @@
+/*
+Invocation:
+
+
+Invocation; note that the number of processes spawned is four times
+the number of cores -- this helps in two ways:
+
+1. Keeping cores busy while I/O operations are in flight
```

Comment:
Interesting so far I have not seen nix evaluation to be significantly I/O bound.

Reply:
This may be specific to my setup.

The machine from which I launch builds has *two* Nix stores: a fast 3TB nvme `/nix/store` and a huge raid-0 spinning-rust secondary store (which is a `trusted-substituter`).  I never run GC on the spinning-rust store.

I want to switch to a single store using bcachefs, but I'm still testing it out on less critical applications.